lib/utils.js

```diff
@@ -54,3 +55,17 @@ export function log(...message: Array<any>) {
     console.debug("Hydrogen:", ...message);
   }
 }
+
+export function renderDevTools() {
+  if (atom.inDevMode() || atom.config.get("Hydrogen.debug")) {
+    try {
+      const devTools = require("mobx-react-devtools");
+      const div = document.createElement("div");
+      document.getElementsByTagName("body")[0].appendChild(div);
+      devTools.setLogEnabled(true);
+      ReactDOM.render(<devTools.default noPanel />, div);
```

Comment:
By default `mobx-react-devtools` come with a small floating panel. `noPanel` disables it.